format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
The only way I can see to support this with our current behavior would be to normatively state that the first index of an array version of `href` is the primary. That would allow us to retain the current behavior while allowing future expansion. If @dgeb is amenable to this, I will attempt to revise this PR later today.